gamemodes/sffa.pwn

```diff
@@ -941,20 +948,20 @@ public OnGameModeInit()
 
 
 	if(!fexist(SETTINGS_FILE))
-		file_Create(SETTINGS_FILE);
+		fopen(SETTINGS_FILE, io_write);
 
 	else
 	{
-	    file_Open(SETTINGS_FILE);
-		file_GetStr("motd", gMessageOfTheDay);
-	    file_Close();
+	    ini_open(SETTINGS_FILE);
+		ini_getString("motd", gMessageOfTheDay);
```

Comment:
woah this needs some indentation fixes, damn pawno!

Reply:
It's pawno doing funky things. Really ancient issue.